service/Abstractions/Models/DataPipelineStatus.cs

```diff
@@ -12,15 +12,10 @@ public class DataPipelineStatus
     [JsonPropertyName("completed")]
     public bool Completed { get; set; } = false;
 
-    [JsonPropertyOrder(1)]
-    [JsonPropertyName("failed")]
-    public bool Failed { get; set; } = false;
-
     /// <summary>
     /// Currently used to track deleted documents.
-    /// TODO: replace with "isDeleting" and "Deleted"
     /// </summary>
-    [JsonPropertyOrder(2)]
+    [JsonPropertyOrder(1)]
```

Comment:
```suggestion
    [JsonPropertyOrder(2)]
```

we can leave this unchanged, no impact